projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -648,15 +669,20 @@ public function render_wpcom_invite_checkbox( $type ) {
 				<table class="form-table">
 					<tr class="form-field">
 						<th scope="row">
-							<label for="invite_user_wpcom"><?php esc_html_e( 'Invite user:', 'jetpack' ); ?></label>
+							<label for="invite_user_wpcom"><?php esc_html_e( 'Invite user', 'jetpack' ); ?></label>
 						</th>
 						<td>
 							<fieldset>
 								<legend class="screen-reader-text">
 									<span><?php esc_html_e( 'Invite user', 'jetpack' ); ?></span>
 								</legend>
 								<label for="invite_user_wpcom">
-									<input name="invite_user_wpcom" type="checkbox" id="invite_user_wpcom" checked>
+									<input 
+										name="invite_user_wpcom" 
+										type="checkbox" 
+										id="invite_user_wpcom" 
+										<?php echo ! class_exists( 'WooCommerce' ) ? esc_attr( 'checked="checked"' ) : ''; ?>"
```

Comment:
WordPress has a [`checked()`](https://developer.wordpress.org/reference/functions/checked/) function that could be used here as well